pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "github-to-sops"
+version = "0.1"
+description = ""
```

Comment:
Let's add this

Reply:
Good catch. Description added as of b962ad31b3091c43e4c22a0bd4fc83262debc776:
![image](https://github.com/tarasglek/github-to-sops/assets/78163326/25a99e69-dc4e-4353-b348-5eea6408296d)